noWord/common/DefaultTransformation.py

```diff
@@ -0,0 +1,74 @@
+
+
+def merge( input, params, context):
+
+    targetResource = {}
+
+    if isinstance( input, list):
+        for resourceName in block['resources']:
+            data = context.getResource(context.resources, resourceName)
+            targetResource.update(deepcopy(data))
+
+    return targetResource
+
+def slice( input, params, context):

```

Comment:
A deep copy would be recommended as the output of the slice transformation still references the original elements.

Reply:
DONE